packages/zoe/src/contracts/priceAggregatorChainlink.js

```diff
@@ -684,7 +686,7 @@ const start = async zcf => {
           pushResult({
             roundId: _roundIdRaw,
             data: _submissionRaw,
-          });
+          }).catch(console.error);
```

Comment:
I'm not confident that I understand why this is OK. Add a comment, perhaps?

Reply:
I'm trying not to change the semantics of the chainlink contract (unused at this time).  Before it just dropped uncaught rejections on the floor (which really needs to be fixed in our `xsnap` module), but in the course of debugging I wanted to log them.